eahub/localgroups/views.py

```diff
@@ -1,14 +1,38 @@
 import json
-from datetime import datetime 
+from datetime import datetime
 
 from django.urls import reverse_lazy
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-
+from django.core.mail import mail_managers
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib import messages
+from django.template.loader import render_to_string
 from .models import LocalGroup as Group
 
 def view_group(request, slug):
     group = get_object_or_404(Group, slug=slug)
+
     return render(request, 'eahub/group.html', {
         'group': group
     })
+
+@login_required(login_url=reverse_lazy('login'))
+def claim_group(request, slug):
+    group = get_object_or_404(Group, slug=slug)
+    if request.method == 'POST':
+        subject = "EA Group claimed: {0}".format(group.name)
+        user_eahub_url = "https://{0}/profile/{1}".format(get_current_site(request).domain,request.user.profile.slug)
+        message = render_to_string('emails/claim_group.html', {
+            'user_eahub_url': user_eahub_url,
+            'user_name': request.user.profile.name,
+            'group_name': group.name,
+            'user_email': request.user.email
+        })
+        mail_managers(subject, message)
+        messages.success(
+            request,
+            ''' Thank you, we have received your request to claim this group.
+            Our admin team will send you an email once they have checked your request. ''',
+        )
+        return redirect('/group/{}'.format(group.slug))
```

Comment:
Same re: `get_absolute_url`

Reply:
Do you mean get_current_site? How should I do it instead?